controllers/news-controller.js

```diff
@@ -38,4 +38,19 @@ const { selectTopics } = require('../models/topics-model');
     .then((articles) => {
         res.status(200).send({ articles });
     })
+  };
+
+  exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+    const promises = [selectCommentsByArticleId(article_id)]
+    
+    if(article_id) {
```

Comment:
in order to get into this endpoint, article id must be defined - the only time this would fail is if article id is any other falsy value in which case psql will handle this for us by telling us we cant successfully query here! so do we need the condition?